public/css/global.css

```diff
@@ -33,12 +33,21 @@ h2 {
 .edit-unclickable {
     color: #303030;
 }
+/*Smaller input field box*/
+.input-sm {
+    width: 30%;
+}
 
 /*Background Color for Table/Form Areas*/
 .lt-grey {
     background-color: #eaeaea ;
 }
 
+.scroll-modal{
+    max-height: 600px;  
+    overflow-y: auto;
+}
+
```

Comment:
I would avoid adding custom css classes since we can use tailwind. check the other comment for a suggestion on how to do the same.

Reply:
Thanks Andrew, I made everything tailwind.